test/descriptor.dart

```diff
@@ -48,8 +49,8 @@ FileDescriptor outOfDateSnapshot(String name) =>
 ///
 /// [contents] may contain [Future]s that resolve to serializable objects,
 /// which may in turn contain [Future]s recursively.
-Descriptor pubspec(Map<String, Object> contents) =>
-    file('pubspec.yaml', yaml(contents));
+PubspecDescriptor pubspec(Map<String, Object> contents) =>
```

Comment:
Could we just return a `Descriptor`, given that `PubspecDescriptor` is an instance of `Descriptor`?